### ISSUE TITLE
Option missing to work with gcloud configurations other than `default`

### ISSUE THREAD

**Issue by headcr4sh** [google-github-actions/setup-gcloud#315]
### TL;DR
My use-case involves working with more than one gcloud configuration within a single GitHub job.
I am therefore looking for a way to specify the "configuration name" when setting up cloud credentials.

### Design

**Action YAML**
```yaml
- name: Set up Cloud SDK (creds-a)
  uses: google-github-actions/setup-gcloud@master
  with:
    configuration_name: config-a
    service_account_key: ${{ secrets.GOOGLE_BACKEND_CREDENTIALS_A }}
    export_default_credentials: false
- name: Set up Cloud SDK (creds-b)
  uses: google-github-actions/setup-gcloud@master
  with:
    configuration_name: config-b
    service_account_key: ${{ secrets.GOOGLE_BACKEND_CREDENTIALS_B }}
    export_default_credentials: false
```

**Resources**
- [gcloud config configurations](https://cloud.google.com/sdk/gcloud/reference/config/configurations)

**Additional information**
I am working on a terraform pipeline where I need one service account key to access the terraform state stored in a GCS bucket and another service account key to perform the actual infrastrucure-as-code/terraform-related tasks

**Comment by bharathkkb** [comment]
While you could do this, I am wondering why use `setup-gcloud` at all. In your pipeline you can set `GOOGLE_BACKEND_CREDENTIALS` for backend auth and `GOOGLE_CREDENTIALS` for provider auth?

**Comment by headcr4sh** [comment]
> While you could do this, I am wondering why use `setup-gcloud` at all. In your pipeline you can set `GOOGLE_BACKEND_CREDENTIALS` for backend auth and `GOOGLE_CREDENTIALS` for provider auth?

Thanks for taking the time to dig into my issue.

You are right in assuming that setting these two environment variables is indeed the solution to my problem at hand.
Therefore, this feature request is not needed by me personally.

If you don't feel that having more than one set of gcloud credentials might be a useful addition to this action, just close this ticket and PR #316  which I have prepared to add the missing (well,... missing not for me as it seems ;-)) functionality.

**Comment by bharathkkb** [comment]
@headcr4sh great! 

> having more than one set of gcloud credentials might be a useful addition to this action

I'll let @averikitsch weigh in. I generally haven't encountered a need to have named gcloud configs in pipelines.

**Comment by averikitsch** [comment]
I agree that this is not needed. It might be nice to add a blurb/solution like this to the README though.
